redisgraph/execution_plan.py

```diff
@@ -0,0 +1,61 @@
+class Operation:
+    def __init__(self, name, args=None):
+        self.name = name
+        self.args = args
+        self.children = []
+
+    def append_child(self, child):
+        self.children.append(child)
+        return self
+
+    def __eq__(self, o: object) -> bool:
```

Comment:
This `Equals` method performs recursion, I think this should be handled by `Execution-Plan` `__eq__` logic,
an Operation is equals to another Operation if its `name` and `args` are the same. I think it can be debatable if we need to require `args` equality.